birthdays-project/BirthdayApp/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
```

Comment:
[Module names should by all lower case](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)

rename to `birthday_app`